cmd/juno/juno.go

```diff
@@ -123,6 +125,7 @@ const (
 	defaultCorsEnable               = false
 	defaultVersionedConstantsFile   = ""
 	defaultPluginPath               = ""
+	defaultLogPort                  = 0
```

Comment:
What do you think about setting it e.g. to 9091 by default instead of random? I guess it would keep things more predictable.

Reply:
0 doesn't mean random here, it acutally means that it's disabled :)

Reply:
See https://github.com/NethermindEth/juno/pull/2245/files/55eb191d431b1e1e536555cae68c7b49557545cf#diff-375d57e386f20eaa5f09f02bb9d28bfc48ac3dca18d0325f59492208219e5618R240

Reply:
ohh, gotcha, thanks